nav2_mppi_controller/include/nav2_mppi_controller/motion_models.hpp

```diff
@@ -52,6 +54,25 @@ class MotionModel
     */
   virtual ~MotionModel() = default;
 
+  /**
+    * @brief Configure motion model on bringup and load required parameters
+    * @param name Name of plugin
+    * @param param_handler Parameter handler object
+    */
+  void on_configure(
+    const std::string & name,
+    ParametersHandler * param_handler)
+  {
+    auto getParam = param_handler->getParamGetter(name);
+    getParam(accel_constraints_.ax_max, "ax_max", 2.0);
```

Comment:
If added to ControlConstraints, then this can/should be removed to that location where the other ControlConstraints are populated. The motion model constructor can take in a pointer to the constraints object (so that values contained can be updated). Each motion model derived class can then decide if it wants to use it, but they all have access to it.

Reply:
Instead of passing the constraints in the constructor, we can define a new function initialize(). Because if we pass it through the constructor, we will need to modify default constructors of DiffDriveMotionModel and OmniMotionModel. Similarly, explicit constructor of AckermannMotionModel takes parameter_handler as the parameter, we will need to modify that also. what is your suggestion this?